WinUIGallery/WinUIGallery.DesktopWap.csproj

```diff
@@ -28,7 +28,8 @@
        in the WinUI repo, or the next ItemGroup below when standalone. -->
   <ItemGroup>
     <PackageReference Include="ColorCode.Core" />
-    <PackageReference Include="CommunityToolkit.Labs.WinUI.SettingsControls" Version="0.0.16" />
+    <PackageReference Include="CommunityToolkit.Labs.WinUI.SegmentedControl" Version="0.0.2" />
+    <PackageReference Include="CommunityToolkit.Labs.WinUI.SettingsControls" Version="0.0.17" />
```

Comment:
Is this needed for this PR?

Reply:
Whoops, pressed the Update all button I guess. Fixed!